### DISCUSSION TITLE
WHY NOT SUPPORT PYTORCH==1.4.0 !!!

### DISCUSSION THREAD

**Discussion by shawnthu** [Lightning-AI/pytorch-lightning#9543] [General]

**Comment by awaelchli** [comment]
Lightning not supporting 1.4.0 means we do not run tests against this version and we do not fix bugs or maintain support for features dropped above torch 1.4. But you should still be able to use 1.4 and it will probably be fine for most uses. Have you tried it?

**Comment by awaelchli** [comment]
Furthermore, if you need to run older projects and make them reproducible, I recommend reverting back to a PL version that still had support for torch 1.4.

**Reply by shawnthu** [reply]
> Furthermore, if you need to run older projects and make them reproducible, I recommend reverting back to a PL version that still had support for torch 1.4.

I'll give a feedback here if have any try on torch version 1.4 in the future
